python-avd/scripts/build-schemas.py

```diff
@@ -24,7 +24,7 @@
 from ansible_collections.arista.avd.plugins.plugin_utils.schema.avdtojsonschemaconverter import AvdToJsonSchemaConverter
 from ansible_collections.arista.avd.plugins.plugin_utils.schema.store import create_store
 
-FRAGMENTS_PATTERN = "*.schema.yml"
+FRAGMENTS_PATTERN = "*.yml"
```

Comment:
not sure if this impact some documentation as well

Reply:
I don't think we specify the file names anywhere, and clearly it leads to missing docs.

Reply:
```suggestion
# Schemas should be named '*.schema.yml', but we include all yml files since we are humans.
FRAGMENTS_PATTERN = "*.yml"
```